tests/Unit/Services/Gateways/DebitCardTest.php

```diff
@@ -54,6 +56,14 @@ public function testAvailabilityWithLocalCurrency()
         $this->assertAvailableForCountries($gateway, [
             Country::MEXICO,
         ]);
+
+        $gateway = new DebitCard(new Config([
+            'baseCurrency' => Currency::UYU,
+        ]));
+
+        $this->assertAvailableForCountries($gateway, [
+            Country::URUGUAY,
+        ]);
     }
```

Comment:
In order to do not break the ["single assert rule"](https://twitter.com/unclebobmartin/status/1078695335935979520) I suggest you to create a test case generator as an alternative for this approach.

Reply:
Provider implemented.
Thank you for the suggestion!